edm4eic.yaml

```diff
@@ -275,8 +282,23 @@ datatypes:
       - int32_t           sector            // The sector this hit occured in
       - edm4hep::Vector3f local             // The local position of the hit in detector coordinates (relative to the sector) [mm]
 
+  edm4eic::CherenkovParticleID:
+    Description: "Cherenkov detector PID"
+    Author: "A. Kiselev, C. Chatterjee, C. Dilks"
+    Members:
+      - int32_t           radiator          // Radiator number
```

Comment:
Unsigned? Presumably part of cellID?

Reply:
Yes, should be unsigned. I don't think we can make it a part of the cell ID, since it's not clear which radiator a pixel hit came from, and in some cases, 2 photons could hit a single pixel, each originating from a different radiator.